### ISSUE TITLE
Cannot load .md files (addon-notes) with preset-create-react-app

### ISSUE THREAD

**Issue by elvorad** [storybookjs/storybook#9446]
### Summary

When I try to use the addon `notes` with Storybook 5.3.0 or 5.3.3 (with preset-create-react-app 1.5.1), it only shows this error message.

![image](https://user-images.githubusercontent.com/29063703/72357154-c8f57c80-36ea-11ea-9b54-90b4f15b9637.png)

When I remove `@storybook/preset-create-react-app` from `main.js`, the Markdown will show. However, Storybook will protest that the preset is missing in the command line:

![image](https://user-images.githubusercontent.com/29063703/72356729-07d70280-36ea-11ea-92d4-499b88c0e10f.png)

### Steps to reproduce

1. Create an empty "create-react-app" folder and enter it.
2. Install Storybook by running `npx -p @storybook/cli sb init`
3. Install `addon-notes` in yarn; add it to `main.js`:

```js
module.exports = {
  stories: ['../src/**/*.stories.js'],
  addons: [
    '@storybook/preset-create-react-app',
    '@storybook/addon-actions',
    '@storybook/addon-notes',
    '@storybook/addon-links',
  ],
};
```

4. Use a markdown file in a story, e.g.
```js
import React from 'react';
import { linkTo } from '@storybook/addon-links';
import { Welcome } from '@storybook/react/demo';
import * as MARKDOWN from './Welcome.md';

5. Create the markdown and run Storybook

export default {
  title: 'Welcome',
  component: Welcome,
  parameters: {
    notes: MARKDOWN
  }
};

export const ToStorybook = () => <Welcome showApp={linkTo('Button')} />;

ToStorybook.story = {
  name: 'to Storybook',
};
```
6. The issue should show once you start the Storybook and try to view the "Notes" section for that story. If you comment out the following line in `main.js`, the markdown will be visible again:
```js
module.exports = {
  stories: ['../src/**/*.stories.js'],
  addons: [
    // '@storybook/preset-create-react-app',
    '@storybook/addon-actions',
    '@storybook/addon-notes',
    '@storybook/addon-links',
  ],
};
```

### Please specify which version of Storybook and optionally any affected addons that you're running

```js
"@storybook/addon-actions": "5.3.0",
"@storybook/addon-links": "5.3.0",
"@storybook/addons": "5.3.0",
"@storybook/preset-create-react-app": "1.5.1",
"@storybook/react": "5.3.0"
```

### Affected platforms 

Latest Firefox and Chrome on macOS Mojave 10.14.6

### Screenshots / Screencast / Code Snippets (Optional)

See above.

### Remark
Please tell me a way how to fix it on my side or provide a bugfix. If something is missing in my report, please let me know. Thanks!

**Comment by eriktoyra** [comment]
I had the exact same issue yesterday and couldn't find a solution. I ultimately switched to using `@storybook/addon-docs` instead of `@storybook/addon-notes` to document my stories in MDX.

**Comment by shilman** [comment]
`Addon-notes` is being superceded by `addon-docs`, which fixes a bunch of bugs and is easier to maintain. Please give it a try! https://medium.com/storybookjs/storybook-docspage-e185bc3622bf

**Comment by robcaldecottvelo** [comment]
We have a lot of `addon-notes` docs and migrating these to MDX/Docs is no mean feat. It would be great to get support for `addon-notes` in 5.3 to give us time to migrate.

**Comment by elvorad** [comment]
I agree with @robcaldecottvelo. I know about `addon-docs`, but we did not yet have time to migrate all the stories yet. I guess we should though. Is `addon-notes` already deprecated and no longer maintained?

**Comment by shilman** [comment]
PR's are very welcome for `addon-notes`! I'll be happy to review, merge, and release them in a timely manner 👍

**Comment by shilman** [comment]
@mrmckeb did something change in the CRA preset to cause it to no longer load `.md` files in 5.3?

**Comment by mrmckeb** [comment]
@shilman CRA never handled `.md` files, so this may have been an oversight as we don't have test cases fo this. We do test for `mdx` with Docs.

Does addon-notes add a loader for this?

**Comment by shilman** [comment]
@mrmckeb No, I think it might be part of Storybook's common loaders? Or `@storybook/react`?

**Comment by mrmckeb** [comment]
Will look, thanks!

**Comment by mrmckeb** [comment]
Correct, that would be a good temporary fix methinks. If you can share what works, I can look to add that to the fix to the preset (I can't look until tomorrow sorry @robcaldecottvelo).

**Comment by robcaldecottvelo** [comment]
It looks like even with the react preset there is definitely still a raw loader for .md files added to the webpack config so I am not sure how to tackle this now.

**Comment by robcaldecottvelo** [comment]
The issue is in `processCraConfig.ts`. I forced the `isStorybook530` to `false` and that fixes it. I don't know why that fixes it yet: I guess the rules are being manipulated somehow but the `raw-loader` one probably needs to be ignored.

**Comment by robcaldecottvelo** [comment]
Maybe this will work:

```js
module.exports = {
  addons: [
    {
      name: '@storybook/preset-create-react-app',
      options: {
        craOverrides: {
          fileLoaderExcludes: ['md'],
        },
      },
    },
  ],
};
```

**Comment by elvorad** [comment]
@robcaldecottvelo: This workaround eliminates the problem, I can confirm it.

**Comment by mrmckeb** [comment]
Thanks, I'll add that to the excludes list for Storybook 5.3.0! I'll get that into the PR I'm working on now.
